api/api_test.go

```diff
@@ -14,6 +15,29 @@ const (
 	mockSplitioAPIURI = "https://mock.sdk.split.io/api"
 )
 
+type mockHandler struct {
+	sync.Mutex
+	count int
+}
+
+func (h *mockHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	h.Lock()
```

Comment:
I am curious about why we are using mutex. Are we having multiple goroutines accessing `h.count`?

Reply:
oh yeah you are right, the mock one won't have multiple goroutines, I'll remove it.